README.md

```diff
@@ -1,12 +1,35 @@
-This is a "fork" of the [Grafana InfluxDB plugin](https://github.com/grafana/grafana/tree/master/public/app/plugins/datasource/influxdb) with changes applied to function with the Heroic time-series database.
+# spotify-heroic-datasource
 
-This is intended to be an improvement on the [previous Heroic datasource](https://github.com/udoprog/udoprog-heroic-datasource) that includes a query builder, table support, variable support, and annotation support.
+[![Build Status](https://travis-ci.org/spotify/spotify-heroic-datasource.svg?branch=master)](https://travis-ci.org/spotify/spotify-heroic-datasource)
 
-# Development Status
+![heroic-datasource-screenshot](https://github.com/spotify/spotify-heroic-datasource/raw/master/heroic-datasource-screenshot.png "Heroic Datasource Plugin")
 
-Beta
```

Comment:
Worth keeping this section and changing Beta to "Active Development" or whatever so folks know what to expect from this?

Reply:
We are about to cut a 1.0 release and mark this GA.